example/src/screens/MultilineWebhookPaymentScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import {
+  Alert,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+  Switch,
+  Platform,
+} from 'react-native';
+import { CardForm, useConfirmPayment } from '@stripe/stripe-react-native';
+import Button from '../components/Button';
+import PaymentScreen from '../components/PaymentScreen';
+import { API_URL } from '../Config';
+import { colors } from '../colors';
+
+export default function MultilineWebhookPaymentScreen() {
+  const [email, setEmail] = useState('');
+  const [saveCard, setSaveCard] = useState(false);
+  const [isComplete, setComplete] = useState(false);
+
+  const { confirmPayment, loading } = useConfirmPayment();
+
+  const fetchPaymentIntentClientSecret = async () => {
+    const response = await fetch(`${API_URL}/create-payment-intent`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email,
+        currency: 'usd',
+        items: [{ id: 'id' }],
+        // request_three_d_secure: 'any',
+      }),
+    });
+    const { clientSecret } = await response.json();
+
+    return clientSecret;
+  };
+
+  const handlePayPress = async () => {
+    // 1. fetch Intent Client Secret from backend
+    const clientSecret = await fetchPaymentIntentClientSecret();
+
+    // 2. Gather customer billing information (ex. email)
+    const billingDetails: PaymentMethodCreateParams.BillingDetails = {
+      email: 'email@stripe.com',
+      phone: '+48888000888',
+      addressCity: 'Houston',
+      addressCountry: 'US',
+      addressLine1: '1459  Circle Drive',
+      addressLine2: 'Texas',
+      addressPostalCode: '77063',
+    }; // mocked data for tests
+
+    // 3. Confirm payment with card details
+    // The rest will be done automatically using webhooks
+    const { error, paymentIntent } = await confirmPayment(clientSecret, {
+      type: 'Card',
+      billingDetails,
+      setupFutureUsage: saveCard ? 'OffSession' : undefined,
+    });
+
+    if (error) {
+      Alert.alert(`Error code: ${error.code}`, error.message);
+      console.log('Payment confirmation error', error.message);
+    } else if (paymentIntent) {
+      Alert.alert(
+        'Success',
+        `The payment was confirmed successfully! currency: ${paymentIntent.currency}`
+      );
+      console.log('Success from promise', paymentIntent);
+    }
+  };
+
+  return (
+    <PaymentScreen>
+      <TextInput
+        autoCapitalize="none"
+        placeholder="E-mail"
+        keyboardType="email-address"
+        onChange={(value) => setEmail(value.nativeEvent.text)}
+        style={styles.input}
+      />
+      <CardForm
+        autofocus
+        cardStyle={inputStyles}
+        dangerouslyGetFullCardDetails
+        style={styles.cardField}
+        placeholder={{
+          cvc: '99',
+          expiration: '00',
+          number: '0231',
+          postalCode: '999',
+        }}
+        onCardComplete={(cardDetails) => {
```

Comment:
Can we rename to `onFormComplete` to make it clear the whole form has to be complete?